internal/constellation/helm/charts/edgeless/constellation-services/charts/autoscaler/templates/coredns-pdb.yaml

```diff
@@ -0,0 +1,10 @@
+apiVersion: policy/v1
+kind: PodDisruptionBudget
+metadata:
+  name: coredns-pdb
+  namespace: {{ .Release.Namespace }}
```

Comment:
It might be better to hard-code `kube-system` here, since CoreDNS is not deployed by Helm and thus the release namespace does not apply.